app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticPagesController < ApplicationController
+	before_action :redirect_to_activity_if_signed_in
```

Comment:
Sir, this code is good in my viewpoint.
Why? When you go to facebook, you see this:


but you don't see this screen anymore when the user is logged in, despite
you enter the same url: facebook.com

This is a good experience since the user doesn't even want to see this
screen which is shown at the landing page once he is logged in.

Do let me know your views.
​

On Wed, May 20, 2015 at 4:24 PM, B V Satyaram <notifications@github.com>
wrote:

> In app/controllers/static_pages_controller.rb
> <https://github.com/chi6rag/HourGlass/pull/1#discussion_r30689845>:
>
> > @@ -0,0 +1,6 @@
> > +class StaticPagesController < ApplicationController
> > +	before_action :redirect_to_activity_if_signed_in
>
> Does this mean that the URL changes when a loggedin user visits root path?
> If so, that is a bad experience.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chi6rag/HourGlass/pull/1/files#r30689845>.
>